.travis.yml

```diff
@@ -9,3 +9,7 @@ after_success: ./gradlew jacocoTestReport coveralls
 
 notifications:
   email: false
+
+env:
+  secure: "B22wUbu/wuuyqMDbNYs3FCNsmcuE0Ji7Yny9yioysr1QxcKgCz0xafOSh1O5aROuKlxXb96NovwBecehJbjtZUnfbqTqaXodt7XG1Wn8FU72TaNfplVPSx2yVPUYU0gTFSloxYVbI+Q6L9JcDsLzqtYeToGB66CLvM7jWmkQWBSjb/LZab3vnSsT65lHvcxY7XO0O+njt73Qbetn+Dw5lztaF8SJk1U5Ho5OzYxjnh7Dyqr+Y7vyE9qFddbT5ZUmieNJmIMeVF/8kEk2jsUBcJTTWGpYuxp92+RaG2Ul9rZlwVzXSJAJBxAnlvYXQfmG3xuDrf1NRsl6QhZquIDS8Xk7qqDK4U4rNWRGroad8yx2I1SrQelxEAEIFxPMS0p713/xGbKdB0GVbl5Zqm3OuUjoaTlztRu6sIanL/nXbJydFpgcevyy3cnw9xb96KxHYe4+sGvsgAbQ32kBnRVxG0D7z9UtvTG0UfwLFxnHaxQB89N/IXPSkNs6FEip81gZwmVd51IIy84FAa4iPoCiNn9eU3c2aSg4XDt/g9AH+jih4rI6t82hvtwAwWNeYKqkMaiXPUIznMgjjS8l3o+iIPae85mqjyNUvNIOzT5+XBWF4G1Y8ncggTzrjb7BrO50XTAOeQMfJ88OMgMxNenYFmJV0lh7gwFm0p7h9cHu/QU="
+  secure: "lq31uKdO5F3IfUsfcmPIvmXkp2BOz56pjOAQ4S1RqHF4LoVH6LKMMd3Ocn0C9GyQfomM5NlKTRogFdJvB2iS6piXHej5iSo2/r5tmPxgX3BVMZwV6u9hEuE58U2sA+/NNwWZtivs+1U6XSRNwA2vi2qB8B5lePifCZjxsqL8l1doD4pwH+uDxXoYktberIAKizAzZFi03RvIaXHgPQYBADwrSYY4kbmrl1fL/jfdZx/ouSD055veIVJqQUGwsrdiq5lgtLSR8UfDzWZLQwDTiMLau7fJdE40lwPBAwpwDHPLdf1IGQfMih8y3YC4bQBNMi1w7J/T1nOpABvD8EZhHypYOHO/aYgWMmJM/EEzcytS52r1fieNoV3+GxxdSbOd6DN2D+hrGnZ34c3sQ8UZWXv6/PZtBBIcOsw132cQ35dOSXx78khg129IauW+RiQ5bRglZMr0Su6Pz3AL7az1CG0IDzzlJC+vmcobPLDTHJWpXIO2dyv/NyzfD0d5tYc9I1Y2/KeLUBxEm6x2GZw/dSQRih9YsZc7FXTyReNE5TQ8qGrDhpKGswr4c8MxW/3Dyktb0VwO2TlCfXwB0QKyTdd5QFSNYZ6OJyn1uPY/jHpTEzYxGAIVVFbWBwyv6NkjsIuFzhyLaOtmp6klHTdqcdFSExtIpL+8QoRmvWM06dc="
```

Comment:
so we need 2 lines with the same yml key?   And this can't be set up in the travis GUI (I put the code climate repo token in the settings in travis for the projects that use code climate)